src/collectors/prometheus_rabbitmq_core_metrics_collector.erl

```diff
@@ -55,44 +55,44 @@
-        {3, channel_get, counter, "Messages fetched with basic.get in automatic acknowledgement mode"},
-        {4, channel_messages_delivered_ack, counter, "Messages delivered to consumers in manual acknowledgement mode"},
-        {5, channel_messages_delivered, counter, "Messages delivered to consumers in automatic acknowledgement mode"},
-        {6, channel_messages_redelivered, counter, "Messages redelivered to consumers"},
-        {7, channel_messages_acked, counter, "Messages acknowledged by consumers"},
-        {8, channel_get_empty, counter, "Number of times basic.get operations fetched no message"}
+        {2, channel_get_ack_total, counter, "Total number of messages fetched with basic.get in manual acknowledgement mode"},
+        {3, channel_get_total, counter, "Total number of messages fetched with basic.get in automatic acknowledgement mode"},
+        {4, channel_messages_delivered_ack_total, counter, "Total number of messages delivered to consumers in manual acknowledgement mode"},
+        {5, channel_messages_delivered_total, counter, "Total number of messages delivered to consumers in automatic acknowledgement mode"},
+        {6, channel_messages_redelivered_total, counter, "Total number of messages redelivered to consumers"},
+        {7, channel_messages_acked_total, counter, "Total number of messages acknowledged by consumers"},
+        {8, channel_get_empty_total, counter, "Total number of times basic.get operations fetched no message"}
     ]},
 
     {connection_churn_metrics, [
-        {2, connection_open, counter, "Connections opened"},
-        {3, connection_close, counter, "Connections closed or terminated"},
-        {4, channel_open, counter, "Channels opened"},
-        {5, channel_close, counter, "Channels closed"},
-        {6, queue_declare, counter, "Queues declared"},
-        {7, queue_create, counter, "Queues created"},
-        {8, queue_delete, counter, "Queues deleted"}
+        {2, connection_open_total, counter, "Total number of connections opened"},
+        {3, connection_close_total, counter, "Total number of connections closed or terminated"},
+        {4, channel_open_total, counter, "Total number of channels opened"},
+        {5, channel_close_total, counter, "Total number of channels closed"},
+        {6, queue_declare_total, counter, "Total number of queues declared"},
+        {7, queue_create_total, counter, "Total number of queues created"},
+        {8, queue_delete_total, counter, "Total number of queues deleted"}
     ]},
 
     {connection_coarse_metrics, [
-        {2, connection_bytes_in, counter, "Bytes received on a connection"},
-        {3, connection_bytes_out, counter, "Bytes sent on a connection"},
-        {4, erlang_process_reductions, counter, "Erlang process reductions"}
+        {2, connection_bytes_in_total, counter, "Total number of bytes received on a connection"},
```

Comment:
Units go last, so in_bytes_total

Reply:
`connection_in_bytes_total` sounds weird. Maybe `connection_incoming_bytes_total`?

Can you think of a better name?

Reply:
`received` would be one option.